models/fates/inst/create_1x1_ref_case.sh

```diff
@@ -102,7 +107,7 @@ echo "*** Running case.setup ***"
 
 cat >> user_nl_clm << \EOF
 hist_empty_htapes = .true.
-hist_fincl1='EFLX_LH_TOT','TSOI_10CM','QVEGT','NEP','GPP','AR','ED_bleaf','ED_biomass','NPP_column’,'NPP','MAINT_RESP','GROWTH_RESP'
+hist_fincl1='EFLX_LH_TOT','TSOI_10CM','QVEGT','NEP','GPP','AR','ED_bleaf','ED_biomass','NPP_column','NPP','MAINT_RESP','GROWTH_RESP'
```

Comment:
Ryan told me that NPP is the output to use, and that NPP_column wasn't useful but I don't remember why.  NEP can stil cause PIO write errors and I would recommend we don't set that until we can confirm that NEP wont crash runs